### PR TITLE
[W5.11][W11-A1] Sashankh Chengavalli Kumar

### PR DESCRIPTION
**Pull request by ChengSashankh**
Enhanced find to make it case-insensitive. Updated help, documentation and code to reflect changes.

### CODE REVIEW

**Comment by wzneo** [comment]
Great job! I like how you extracted the method for lowerCase.

File: src/seedu/addressbook/commands/FindCommand.java [hunk 1]
```diff
@@ -40,17 +40,27 @@ public CommandResult execute() {
         return new CommandResult(getMessageForPersonListShownSummary(personsFound), personsFound);
     }
 
+    private Set<String> getSetInLowerCase(Set<String> wordSet){
```
Comment (okkhoy) [inline src/seedu/addressbook/commands/FindCommand.java:43]: Can you think of a more appropriate method name? (Hint: `get<What>SetInLowerCase`?)

File: src/seedu/addressbook/commands/FindCommand.java [hunk 2]
```diff
@@ -40,17 +40,27 @@ public CommandResult execute() {
...
+        Set<String> lowerCaseSet = new HashSet<String>();
+        for(String word : wordSet){
```
Comment (okkhoy) [inline src/seedu/addressbook/commands/FindCommand.java:45]: need a space between `) {`

File: src/seedu/addressbook/commands/FindCommand.java [hunk 3]
```diff
@@ -40,17 +40,27 @@ public CommandResult execute() {
...
      * @return list of persons found
      */
     private List<ReadOnlyPerson> getPersonsWithNameContainingAnyKeyword(Set<String> keywords) {
+        Set<String> keywordsInLowerCase = getSetInLowerCase(keywords);
         final List<ReadOnlyPerson> matchedPersons = new ArrayList<>();
         for (ReadOnlyPerson person : addressBook.getAllPersons()) {
             final Set<String> wordsInName = new HashSet<>(person.getName().getWordsInName());
-            if (!Collections.disjoint(wordsInName, keywords)) {
+            final Set<String> lowerCaseWordsInName = getSetInLowerCase(wordsInName);
+            if (!Collections.disjoint(lowerCaseWordsInName, keywordsInLowerCase)) {
```
Comment (okkhoy) [inline src/seedu/addressbook/commands/FindCommand.java:63]: Is there a better way to do this than convert both to lower case?

**Review by okkhoy — COMMENTED** [review]
:+1:  for a clean PR

**Comment by okkhoy** [comment]
@ChengSashankh 
Some comments added. Please ack & close the PR after reading comments.

**Comment by ChengSashankh** [comment]
Comments noted with thanks.

**Pull request closed without merging**